### DISCUSSION TITLE
Cost management

### DISCUSSION THREAD

**Discussion by tkimmm** [planetscale/discussion#545] [Feature Requests] [answered]
Hi Planetscale,

We're on a few scaler databases at my org and due to pricing model it's imperative that we keep on top of usage metrics.

Right now I can't see anything in the API docs that seems to suggest it's possible to receive these metrics through the public API. However I can see that the GUI is rendering the row read/write/data from an internal API. 

Is there any plan to open up this endpoint to the public or do you have any recommendations, I can't seem to find any notification functionality on your platform.

Thanks

**Comment by jgreet** [comment]
Currently the way to access those usage numbers from outside of the application is through our Datadog integration:

https://planetscale.com/docs/integrations/datadog

It's possible we'll add a public API endpoint for usage data in the future.

**Comment by mscoutermarsh** [comment, marked as answer]
Update: Billing data is available via the API: https://planetscale.com/changelog#billing-data-now-available-via-the-api

We also have spend alerts: https://planetscale.com/changelog#monthly-spend-alerts
